src/track/variant/VariantTrack.ts

```diff
@@ -282,7 +282,7 @@ export class VariantTrack<Model extends VariantTrackModel = VariantTrackModel> e
     protected displayLabel(
         variantId: string,
         baseCharacter: string,
-        color: ArrayLike<number>,
+        color: Array<number>,
```

Comment:
Why do we want color to be an "Array" rather than "ArrayLike"?

Reply:
Not sure. It comes from this commit: https://github.com/VALIS-software/valis-hpgv/commit/d54b65319bacd8397055ca3359b4a895e5e7fa83

Reply:
It looks like this is a change George made, to correct a Typescript error, so that seems good.